templates/includes/related-ce-content.html

```diff
@@ -0,0 +1,53 @@
+    <h2>Discover highlights from the collection</h2>
+    <ul class="related-ce-content-group__list">  
+        <li class="col-md-6 related-ce-content">
+            <a href="{% pageurl page.time_period %}">
+                <picture>
+                    {% image page.time_period.teaser_image fill-480x351 as teaser %}
+                    <source media="(max-width: 640px)" srcset="{{ teaser.url }}">
+                    {% image page.time_period.teaser_image fill-480x250 as teaser %}
+                    <source media="(max-width: 768px)" srcset="{{ teaser.url }}">
+                    {% image page.time_period.teaser_image fill-330x250 as teaser %}
+                    <source media="(max-width: 900px)" srcset="{{ teaser.url }}">
+                    {% image page.time_period.teaser_image fill-315x300 as teaser %}
+                    <source media="(max-width: 991px)" srcset="{{ teaser.url }}">
+                    {% image page.time_period.teaser_image fill-435x367 as teaser %}
+                    <source media="(max-width: 1200px)" srcset="{{ teaser.url }}">
+                    {% image page.time_period.teaser_image fill-525x367 as teaser %}
+                    <source media="(max-width: 1440px)" srcset="{{ teaser.url }}">
+                    <img class="related-ce-content__image" src="{{ teaser.url }}" alt="" />
+                </picture>
+                <div class="related-ce-content__summary">
+                    <h3 class="no-margin-top">{{ page.time_period }}</h3>
+                </div>
+            </a>
+        </li>
+
+        <li class="col-md-6 related-ce-content">
+            <a href="{% pageurl page.topic %}">
+                <picture>
+                    {% image page.topic.teaser_image fill-480x351 as teaser %}
+                    <source media="(max-width: 640px)" srcset="{{ teaser.url }}">
+                    {% image page.topic.teaser_image fill-480x250 as teaser %}
+                    <source media="(max-width: 768px)" srcset="{{ teaser.url }}">
+                    {% image page.topic.teaser_image fill-330x250 as teaser %}
+                    <source media="(max-width: 900px)" srcset="{{ teaser.url }}">
+                    {% image page.topic.teaser_image fill-315x300 as teaser %}
+                    <source media="(max-width: 991px)" srcset="{{ teaser.url }}">
+                    {% image page.topic.teaser_image fill-435x367 as teaser %}
+                    <source media="(max-width: 1200px)" srcset="{{ teaser.url }}">
+                    {% image page.topic.teaser_image fill-525x367 as teaser %}
+                    <source media="(max-width: 1440px)" srcset="{{ teaser.url }}">
+                    <img class="related-ce-content__image" src="{{ teaser.url }}" alt="" />
+                </picture>
+                <div class="related-ce-content__summary">
+                    <h3 class="no-margin-top">{{ page.topic }}</h3>
+                </div>
+            </a>
+        </li>
+    </ul>
+</div>
```

Comment:
So, it turns out the test errors are coming from this template... `page.topic` and `page.time_period` are both optional, so we need some conditionals to account for one or neither being available to display :)

Reply:
Cool, thanks Andy. I've put some conditionals in there now 👍